chat.js

```diff
@@ -948,3 +992,39 @@ Chat.toDurationString = function (number, options) {
 	}
 	return parts.slice(positiveIndex).reverse().map((value, index) => value ? value + " " + unitNames[index] + (value > 1 ? "s" : "") : "").reverse().slice(0, precision).join(" ").trim();
 };
+
+/**
+ * @param  {string} str
+ * @return {string}
+ */
+Chat.md5 = function (str) {
+	 let generator = crypto.createHash('md5');
```

Comment:
If we don't need to be cryptographically secure, why not use crc32? Also, sha256 has hardware acceleration under recent intel chips (newer than skylake)

(crc32 is also hardware accelerated since intel nehalem)